### ISSUE TITLE
Popouts do not work in Chrome (tested on version 88.0.4324.192)

### ISSUE THREAD

**Issue by javidnoutash** [caplin/FlexLayout#186]
Popouts work fine in Firefox and Safari but not in Chrome.
When you click on the popout button of a tab, it does get the "This panel is shown in a floating window" message but the popout is not opened in Chrome (or closed before it's opened). It happens when pop ups are allowed in Chrome.

**Comment by nealus** [comment]
Works for me in chrome (including version 88.0.4324.192 and latest), what operating system are you using? do you get any messages in the developer console?

**Comment by javidnoutash** [comment]
I'm on macOS Catalina v10.15.7

This is what I get in the main window's console:

![Screen Shot 2021-03-18 at 10 59 33 AM](https://user-images.githubusercontent.com/31023251/111544917-1cc19100-87da-11eb-8783-7839630465b7.png)

I don't see the popped out window to check its console.

**Comment by nealus** [comment]
I'm on the same mac version and cannot recreate the problem.

Questions:
1 if you click the "Show Window" are there any other messages in the console?
2 if you select the Chrome Menu bar and then Window, do you see the missing window at the bottom of the menu?
3 Are you using multiple monitors?, or an unusual setup?
4 if you press the "Dock Window" does the content return to the tab?
5 if you use an Incognito window (from the Chrome file menu) do you get the same issue?
6 Can you check your Chrome Preferences for any changes that may effect this.
7 Which Chrome extensions have you got installed?

**Comment by javidnoutash** [comment]
Thanks for following up.

Answers:
1. if you click the "Show Window" are there any other messages in the console?
No, no other messages.
2. if you select the Chrome Menu bar and then Window, do you see the missing window at the bottom of the menu?
No, I don't.
3. Are you using multiple monitors?, or an unusual setup?
Yes, I have two extra monitors connected to my laptop.
4. if you press the "Dock Window" does the content return to the tab?
Yes, it does.
5. if you use an Incognito window (from the Chrome file menu) do you get the same issue?
The popup works fine in Incognito mode.
6. Can you check your Chrome Preferences for any changes that may effect this.
I can't point my finger on anything as such.
7. Which Chrome extensions have you got installed?
I had quite a few extensions installed, but the one causing the popout window not to show was the AdBlocker Pro. I disabled it and the popup works just fine.

I'm happy to keep my AdBlocker disabled, but is there a way to get around it?

**TLDR; AdBlocker Chrome extension was causing the popout window not to show.**

**Comment by nealus** [comment]
If you can find a way to determine if the add blocker is active (or popouts will be disabled) then you can use the supportsPopout prop to prevent FlexLayout from showing the popout icon.
